apps/estuary/lib/estuary/event_table.ex

```diff
@@ -0,0 +1,14 @@
+defmodule Estuary.EventTable do
+  @moduledoc """
+  This modules does the CRUD operation for event_stream in the presto database
```

Comment:
This module currently does NOT do CRUD operation, and it may never do all of CRUD. I'd either do a module doc false or explain the current behavior of the module (just creates the table), otherwise this comment is misleading.

Reply:
That make sense to me, I'll update it to show the actual behavior.